gamestonk_terminal/helper_funcs.py

```diff
@@ -27,6 +28,24 @@
 
 
 def check_valid_path(path: str) -> str:
+    """Argparse type function to test is path is valid
+
+    Parameters
+    ----------
+    path: str
+        Path supplied
+
+    Returns
+    -------
+    path: str
```

Comment:
I think on returns usually you specify the type and what it is. I think the name of the variable usually is unused.

Reply:
```python
Returns
-------
err_code : int
    Non-zero value indicates error code, or zero on success.
err_msg : str or None
    Human readable error message, or None on success.
```
from: https://numpydoc.readthedocs.io/en/latest/format.html

Reply:
Got it, is based on whether or not it is defined!